docs/usage/administration/monthly_active_users.md

```diff
@@ -0,0 +1,73 @@
+# Monthly Active Users
+
+Synapse can be configured to record the number of monthly active users (also referred to as MAU) on a given homeserver.
+For clarity's sake, MAU only tracks local users.
+
+Please note that the metrics recorded by the [Homeserver Usage Stats](/usage/administration/monitoriing/reporting_homeserver_usage_statistics)
+are calculated differently. The `monthly_active_users` does not take into account any of the rules below, and counts any users 
+who have made a request to the homeserver in the last 30 days.
+
+See the [configuration manual](/usage/configuration/config_documentation.html#limit_usage_by_mau) for details on how to configure MAU.
+
+## Calculating active users
+
+Individual user activity is measured in active days, so if a user performs an action on a given day, that day is then recorded. When
+calculating the MAU figure, any users with a recorded action in the last 30 days are considered part of the cohort.
+
+A user is **never** considered active if they are either:
+ - Part of the trial day cohort (described below)
+ - Have an `appservice_id`.
+
+Otherwise, any request to Synapse will mark the user as active. Internally, this is any request that records
+the client's IP address into the database.
+
+The MAU value is recalculated once every 5 minutes for active users, while inactive users are removed from the cohort once every hour.
+Internally this works by checking all users, and adding any recently active users to the `monthly_active_users` table. Every hour, any
+users with timestamps later than 30 days are removed. The sum of all rows of that table is the final count of active users.
+
+It is important to note that **deactivated** users are not immediately removed from the pool of active users, but as blocked users won't
+perform actions they will eventually be removed from the cohort.
+
+### Trial days
+
+If `mau_trial_days` is set, the user must have had activity at least this number of days apart for them to be considered part of the cohort.
+As an example, if `mau_trial_days` is set to `2` and Alice is active on days 1 and 3 then they will be counted as an active user. If Bob
+is active on days 1 and 2, then they will NOT be counted as active. Please note that users do not need to be active on concurrent days to
+be deemed active.  
```

Comment:
Does it matter at all when they registered? (I guess registering is considered activity?)

But if a user is activate on days 1 & 2, then again on day 32 -- are they still active when we check on day 35?

Reply:
> I guess registering is considered activity?

Registering counts as activity yep.

> Does it matter at all when they registered?

Looks like we use the user's `creation_ts` when calculating whether the user is in the trial period or not so it does matter:

https://github.com/matrix-org/synapse/blob/1eaff087af9d6c0985a44e168089cbee0651d9f7/synapse/storage/databases/main/registration.py#L217-L236